docs/DeveloperGuide.adoc

```diff
@@ -775,6 +859,10 @@ Use case resumes at step 2.
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  A user should open the application with the previous entries in a addressbook pre-loaded.
+.  Should not experience visible lag or delays above 0.5s when running any command within the addressbook.
```

Comment:
`0.5s` is pretty arbitrary. Does that mean 0.6s is no good? `Should not experience visible lag` seems sufficient.